ractor_actors/src/net/tcp/mod.rs

```diff
@@ -3,27 +3,14 @@
 // This source code is licensed under both the MIT license found in the
 // LICENSE-MIT file in the root directory of this source tree.
 
-//! This module provides two actors to manage TCP sessions with and without TLS encryption.
-//! We utilize [tokio_rustls] for managing encrypted sessions based on `rustls`. There basic primative
-//! actors supported here are
-//!
-//! 1. `TcpListener` - A server socket listener which accepts incoming requests and calls a callback on new received sessions
-//! 2. [TcpSession] - An actor which manages reading and writing frames to a given socket. Can read and write concurrently through
-//!    an interface actor
-//! 3. [NetworkStream] - Represents either encrypted server or client sockets or an unencrypted socket.
-//! 4. [FrameReceiver] - An implementation of this trait must be provided to the [TcpSession] in order to handle/decode incoming frames of
-//!    messages.
-
-pub mod listener;
-pub mod session;
+//! TCP server and session actors which [Frame] messages
```

Comment:
does this pass `cargo` docs generation?

Reply:
Yes, the comment seems to go in the module overview page. I noticed that the sentence isn't quite complete so I'll fix that.